pages/learn/index.tsx

```diff
@@ -90,6 +93,24 @@ const LearnPage: CustomPage<{
           transition={transition(DURATION.medium4, EASING.standard)}
         >
           <Header>{t("schedule")}</Header>
+          <TextGlance
```

Comment:
Not sure if there should be a way to trigger/logic to show or remove this or not, not for the sake of urgency I will let this pass for now

Reply:
Every time we did a banner like this, we just commented it out when we no longer needed it. Though I agree that moving forward it would be better if banners were server-side.